tests/x11/tracker/tracker_by_command.pm

```diff
@@ -24,19 +24,20 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_leap);
 
 sub run {
     x11_start_program('xterm');
     if (is_sle('<12-SP2')) {
         script_run "tracker-search newfile";
     }
     else {
-        script_run 'tracker search emtpyfile';
+        my $trackercmd = (is_sle('<16') or is_leap('<16.0')) ? 'tracker' : 'tracker3';
+        script_run "$trackercmd search emtpyfile";
         record_soft_failure 'bsc#1074582 tracker can not index empty file automatically' if check_screen 'tracker-cmdsearch-noemptyfile', 30;
```

Comment:
If this soft failure does not apply to TW, then IMO this could be split as:

```
if (is_sle('<12-SP2')) {
...
} elsif (is_sle('<16') || is_leap('<16')) {
...
} else {
...
}
```

Reply:
it still applies (but there is no matching needle; the issue of 'detecting a sofrfail based on a needle matching non-present stuff)

Reply:
erm.. I wonder if that actually happens because of the typo

prep_tracker does touch the correct file name